std/net/isemail.d

```diff
@@ -1839,11 +1826,17 @@ T[] substr (T) (T[] str, ptrdiff_t start = 0, ptrdiff_t length = ptrdiff_t.min)
     return str[start .. end];
 }
 
+///
 unittest
 {
     assert("abcdef".substr(-1) == "f");
     assert("abcdef".substr(-2) == "ef");
     assert("abcdef".substr(-3, 1) == "d");
+}
+
+
```

Comment:
Nitpick: please delete redundant blank line. There should be just 1 empty line between the blocks.